### PR TITLE
Fix: typos in README.md 

### PR DESCRIPTION
**Pull request by omahs**
Fix: typos in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,18 +38,18 @@ A few random points of note:
 
 ### Tor
 
-This website is available natively on Tor. The onion address is in `_includes/onion.html` and a signed document containing the same address is in `/onion.txt`. The address in these 2 files must *always* match. The `onion.txt` files also includes the onion address of the 'downloads' subdomain.
+This website is available natively on Tor. The onion address is in `_includes/onion.html` and a signed document containing the same address is in `/onion.txt`. The address in these 2 files must *always* match. The `onion.txt` files also include the onion address of the 'downloads' subdomain.
```
Comment (plowsof) [inline README.md:41]:
```suggestion
This website is available natively on Tor. The onion address is in `_includes/onion.html` and a signed document containing the same address is in `/onion.txt`. The address in these 2 files must *always* match. The `onion.txt` file also includes the onion address of the 'downloads' subdomain.
```
there is one `onion.txt` file, it includes multiple onion addresses

File: README.md [hunk 2]
```diff
@@ -38,18 +38,18 @@ A few random points of note:
...
 
 ## Translation
 
-In this section you'll find the info you need to translate a page and add a new translation, but keep in mind that Monero has a [Localization Workgroup](https://github.com/monero-ecosystem/monero-translations) who coordinate and give support to translators-volunteers. For live support/request of information, come chat on `#monero-translations` on Matrix or IRC (Libera.chat)
-The entire website is translatable on Weblate, an easy to use localization platform that provide contributors with a user friendly interface: [translate.getmonero.org](https://translate.getmonero.org). Before translating, please read [the guide for translators](https://github.com/monero-ecosystem/monero-translations/blob/master/weblate.md), which contains all the info and workflows you need to know before starting.
+In this section you'll find the info you need to translate a page and add a new translation, but keep in mind that Monero has a [Localization Workgroup](https://github.com/monero-ecosystem/monero-translations) that coordinates and gives support to translators-volunteers. For live support/request of information, come chat on `#monero-translations` on Matrix or IRC (Libera.chat)
```
Comment (plowsof) [inline README.md:51]: original reads fine. i think this is just personal preference / not a typo

File: README.md [hunk 3]
```diff
@@ -247,7 +247,7 @@ Where `ENTRY` is a one word identifier for the title/name of your Moneropedia en
 
 ### 2. Make the localized File
 
-Navigate to the /\_i18n/en/resources/moneropedia folder and make a new file. give it the same <name>.md than in previous step.
+Navigate to the /\_i18n/en/resources/moneropedia folder and make a new file. give it the same <name>.md as in previous step.
```
Comment (plowsof) [inline README.md:250]:
```suggestion
Navigate to the /\_i18n/en/resources/moneropedia folder and make a new file. give it the same <name>.md as in the previous step.
```
reads better?

**Comment by omahs** [comment]
@plowsof Thanks for the review! I just updated

**Review by erciccione — APPROVED** [review]

**Comment by plowsof** [comment]
just needs squashing

**Review by erciccione — COMMENTED** [review]
yep, i missed the two commits. Needs to be squashed into 1 commit before it can be merged

**Comment by omahs** [comment]
Just squashed 👍

**Review by hinto-janai — APPROVED** [review]

**Review by nahuhh — APPROVED** [review]

**Pull request closed without merging**